### DISCUSSION TITLE
[A-L] (2023/24) Foglio 7 - Esercizio 7

### DISCUSSION THREAD

**Discussion by Elia-Belli** [sapienzastudentsnetwork/calcolo-delle-probabilita#187] [Esercizi - Bertini] [answered]
![Screenshot 2023-11-13 194720](https://github.com/sapienzastudentsnetwork/calcprob2324/assets/107944476/f01e0d23-52ae-4706-9efc-9d27c7dab032)

**Comment by IonutCicio** [comment, marked as answer]
Per un evento $A$ sia $1_A$ la variabile aleatoria che rende $1$ se $\omega \in A$ e $0$ se $\omega \in A^{\complement}$, quindi

$$
1_A = 
\begin{cases}
1 & \omega \in A \\
0 & \omega \in A^{\complement}
\end{cases}
$$ 

#### Siano $A$ e $B$ eventi. Verificare che $1_{A^{\complement}} = 1 - 1_A$ e che $1_{A \cap B} = 1_A 1_B$

$$
1 - 1_A = 
\begin{cases}
1 - 1 & \omega \in A \\
1 - 0 & \omega \in A^{\complement}
\end{cases} = 
\begin{cases}
0 & \omega \in A \\
1 & \omega \in A^{\complement}
\end{cases}
= 1_{A^{\complement}}
$$

Per il secondo punto

$$
1_A 1_B = 
\begin{cases}
1 \cdot 1 & \omega \in A \land \omega \in B \\
1 \cdot 0 & \omega \in A \land \omega \notin B\\
0 \cdot 1 & \omega \notin A \land \omega \in B\\
0 \cdot 0 & \omega \notin A \land \omega \notin B \\
\end{cases} = 
\begin{cases}
1 & \omega \in A \land \omega \in B \\
0 & \omega \notin A \lor \omega \notin B
\end{cases} =
\begin{cases}
1 & \omega \in A \land \omega \in B \\
0 & \omega \notin A \lor \omega \notin B
\end{cases} =
$$

Applicando De Morgan

$$
= \begin{cases}
1 & \omega \in A \land \omega \in B \\
0 & \neg (\omega \in A \land \omega \in B)
\end{cases} =
\begin{cases}
1 & \omega \in A \land \omega \in B \\
0 & (\omega \in A \land \omega \in B)^{\complement}
\end{cases} =
1_{A \cap B}
$$

#### Siano $a_1, a_2, ..., a_n, b_n \in \mathbb{R}$, convincersi della validità di 

$$
\prod\limits_{i = 1}^{n} (a_i + b_i) = \sum\limits_{I \subseteq \\{1, ..., n \\} } \prod\limits_{i \in I} a_i \prod\limits_{j \in I^{\complement}} b_j
$$

Prendiamo un esempio, $a_1 = 2, a_2 = 3, b_1 = 1, b_2 = 2$, si avrà

$$
(2 + 1) (3 + 2) = (1 \cdot 2) + (2) (2) + (3) (1) + (2 \cdot 3) \implies 15 = 15
$$

Le parentesi tonde a destra indicano i vari gruppi che si formano per i vari sottoinsiemi $I \subseteq \\{ 1, ..., n \\}$

$$
\begin{cases}
(a_1 \cdot a_2) \cdot 1 & I = \emptyset \\
(a_1) (b_2) & I = \\{ 1 \\} \\
(a_2) (b_1) & I = \\{ 2 \\} \\
1 \cdot (b_1 \cdot b_2) & I = \\{ 1, 2 \\}
\end{cases}
$$

E non bisogna fare altro che la somma fra questi. Si nota _(dall'esempio)_ che questa è la distribuzione binomiale, in cui $a_i$ può essere diverso da $a_j$ per $i \neq j$. Ora che è chiaro cosa succede, si può passare alla dimostrazione per induzione.

#### Caso Base, $n = 1$

$$
a_1 + b_1 = \sum\limits_{I \subseteq \\{1 \\} } \prod\limits_{i \in I} a_i \prod\limits_{j \in I^{\complement}} b_j 
\implies
\begin{cases}
b_1 & I = \emptyset \\
a_1 & I = \\{ 1 \\}
\end{cases}
\implies
a_1 + b_1 = b_1 + a_1
$$

#### Passo Induttivo

Bisogna verificare che 

$$
\prod\limits_{i = 1}^{n} (a_i + b_i) = \sum\limits_{I \subseteq \\{1, ..., n \\} } \prod\limits_{i \in I} a_i \prod\limits_{j \in I^{\complement}} b_j \implies
\prod\limits_{i = 1}^{n + 1} (a_i + b_i) = \sum\limits_{I \subseteq \\{1, ..., n, n + 1 \\} } \prod\limits_{i \in I} a_i \prod\limits_{j \in I^{\complement}} b_j
$$

L'idea sarebbe quella di considerare produttoria fino a $n$ una volta moltiplicandola per $a_{n + 1}$ e l'altra per $b_{n + 1}$ _(che è quello che succede quando si distribuisce un prodotto rispetto alla somma normalmente)._ 

Il che corrisponde a separare i sottoinsiemi per cui $n + 1 \notin I$ da quelli per cui $n + 1 \in I$

$$
\prod\limits_{i = 1}^{n} (a_i + b_i) \cdot (a_{n + 1} + b_{n + 1}) = 
\biggl( \sum\limits_{I \subseteq \\{1, ..., n \\} } \prod\limits_{i \in I} a_i \prod\limits_{j \in I^{\complement}} b_j \biggl) \cdot a_{n + 1} +
\biggl( \sum\limits_{I \subseteq \\{1, ..., n \\} } \prod\limits_{i \in I^{\complement}} a_i \prod\limits_{j \in I} b_j \biggl) \cdot b_{n + 1} \implies
$$

Ora si può usare l'ipotesi induttiva

$$
\implies \prod\limits_{i = 1}^{n} (a_i + b_i) \cdot (a_{n + 1} + b_{n + 1}) = 
\prod\limits_{i = 1}^{n} (a_i + b_i) \cdot a_{n + 1} +
\prod\limits_{i = 1}^{n} (a_i + b_i) \cdot b_{n + 1} = \prod\limits_{i = 1}^{n} (a_i + b_i) (a_{n + 1} + b_{n + 1})
$$

#### Usare i due punti precedenti e le proprietà del valore atteso per dimostrare il principio di inclusione esclusione

L'idea della dimostrazione parte da un'intuizione particolare

$$
\mathbb{P}(\bigcup\limits_{i = 1}^n A_i) = \mathbb{E}(1_{\bigcup\limits_{i=1}^n A_i})
$$

Per spiegare la formula qui sopra bisogna rifarsi alla definizione di $\mathbb{E}(X)$ (valore atteso di X)

$$
\mathbb{E}(X) = \sum\limits_{x \in \Im(X)} x \cdot \mathbb{P}(X = x)
$$

In questo caso, se abbiamo l'insieme $\\{a, b, c, d, e, f\\}$, per rappresentare un sottoinsieme $\\{a, d, e\\}$ si può usare una tabella in cui associo ad ogni lettera 1 o 0 se è presente o meno nel sottoinsieme

| a | b | c | d | e | f |
|--|--|--|--|--|--|
| 1 | 0 | 0 | 1 | 1 | 0 |

A questo punto risulta facile vedere come la probabilità del sottoinsieme ${a, d, e}$ non sia altro che 

$$
1 \cdot \mathbb{P}(1_{\\{a\\}} = 1) + 1 \cdot \mathbb{P}(1_{\\{d\\}} = 1) + 1 \cdot \mathbb{P}(1_{\\{e\\}} = 1) + 0 =
\mathbb{E}(1_{\\{a\\} \cup \\{d\\} \cup \\{e\\}})
$$

Ora si può procedere con la dimostrazione

$$
\mathbb{E}(1_{\bigcup\limits_{i=1}^n A_i}) =  
\mathbb{E}(1 - 1_{{\bigcup\limits_{i=1}^n A_i}^{\complement}}) =
\mathbb{E}(1 - 1_{{\bigcap\limits_{i=1}^n A_i^{\complement}}}) =
$$

$$
\mathbb{E}(1 - \prod\limits_{i=1}^n 1_{A_i^{\complement}}) =
1 - \mathbb{E}(\prod\limits_{i=1}^n 1_{A_i^{\complement}}) =
$$

$$
= 1 - \mathbb{E}(\prod\limits_{i=1}^n (1 - 1_{A_i})) =
= 1 - \mathbb{E}(\sum\limits_{I \subseteq {1, ..., n}} (-1)^{|I|} \prod\limits_{i \in I} 1_{A_i})) =
$$

$$
= 1 - \sum\limits_{I \subseteq {1, ..., n}} (-1)^{|I|} \mathbb{P}(\bigcap\limits_{i \in I} A_i)
$$

1. Passo da $\mathbb{P}$ a $\mathbb{E}$
2. Passo al valore d'attesa del complementare
3. Applico De Morgan 
4. Converto la somma in prodotto per la proprità al punto (1) dell'esercizio
5. Scrivo il complementare di $1_{A_i^{\complement}}$ come $1 - 1_{A_i}$ per il punto (1) dell'esercizio
6. Uso la seconda dimostrazione per passare dal prodotto alla distribuzione 
7. Uso la linearità del valore d'attesa

#noncisareimaiarrivatodasolo
